tests/unit/package_managers/test_js_utils.py

```
@@ -0,0 +1,94 @@
+
+from hermeto.core.errors import PackageRejected
+from hermeto.core.package_managers.javascript.js_utils import (
+    clone_repo_pack_archive,
+    extract_git_url_parts,
+)
+from hermeto.core.rooted_path import RootedPath
+
+
+class TestExtractGitUrlParts:
+    @pytest.mark.parametrize(
+        "url, expected",
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)